index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>product landing page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header id="header">
+        <div id="title">
+            <img id="header-img" alt="HD logo " src="logo.png">
+        </div>
+    <nav id="nav-bar">
+        <a class="nav-link" href="#home">Home</a>
+        <a class="nav-link" href="#video_HD">Video</a>
+        <a class="nav-link" href="#about_us">About-Us</a>
+    </nav>
+</header>
+<main>
+<section id="home" >
+    <h1 class="well"> Welcome to HD Graphics Design</h1>
+    <img class="home" src="s.png" alt="HD Graphics" >
+<p class="hd"> We are here to serve you with new design with different styles.</p>
+    <h2>Some Of Our Design Aid :</h2>
+<br> 
+ <ul>
+<li>photoshop</li>
+<li>Indesign</li>
+<li>and other</li>
+</ul> 
+</section>
+<section id="video_HD">
+    <h2 class="video">Watch our design , how it looks!</h2>
+    <video id="video" controls src="HD-video.mp4">
```

Comment:
The alignment of the h2 tag and video tag would require a change because the alignment position of the two tags is different, which may affect the project how it looks.